#Command indicating which pneumatic valve to open
Header header
int8 command
